jscomp/core/js_dump_program.ml

```diff
@@ -156,9 +160,7 @@ let pp_deps_program
     P.string f empty_explanation 
     (* This is empty module, it won't be referred anywhere *)
   else 
-    let output_dir = Filename.dirname output_prefix in 
-    P.string f L.strict_directive; 
-    P.newline f ;    
+    let output_dir = Filename.dirname output_prefix in   
```

Comment:
I am unclear how does this achieve the desired behavior?

Reply:
Instead of outputting it on all files it's just amd and commonjs